apis/Google.Cloud.Logging.Log4Net/Google.Cloud.Logging.Log4Net/GoogleStackdriverAppender.cs

```diff
@@ -190,6 +203,8 @@ public override void ActivateOptions()
                 Severity = LogSeverity.Warning,
                 LogName = _logName,
                 Resource = _resource,
+                // This does not use patterns in the custom labels if they are enabled.

```

Comment:
I don't really understand this comment. Could you clarify it a bit? Admittedly I don't know about the feature very well...

Reply:
Clarification attempted